src/VisualStudio/CSharp/Test/DocumentOutline/DocumentOutlineTestsBase.cs

```diff
@@ -118,10 +119,25 @@ private async Task<TestLspServer> CreateTestLspServerAsync(EditorTestWorkspace w
             solution = solution.WithAnalyzerReferences([new TestAnalyzerReferenceByLanguage(DiagnosticExtensions.GetCompilerDiagnosticAnalyzersMap())]);
             await workspace.ChangeSolutionAsync(solution);
 
-            var server = await TestLspServer.CreateAsync(workspace, new InitializationOptions(), _logger);
+            var server = await EditorTestLspServer.CreateAsync(workspace, new InitializationOptions(), _logger);
             return server;
         }
 
+        internal class EditorTestLspServer : AbstractTestLspServer<EditorTestWorkspace, EditorTestHostDocument, EditorTestHostProject, EditorTestHostSolution>
```

Comment:
these are the tests that still require the EditorTestWorkspace because they deal with text buffers in the tests.